docs/UPGRADING.md

```diff
@@ -0,0 +1,145 @@
+# 1.0.0 Migration Guide
```

Comment:
Can this be symlinked to the version in the root?

Reply:
fixed this, I think I forgot `-s` when I ran `ln` command.